pkg/cmd/target.go

```diff
@@ -100,6 +106,11 @@ func NewTargetCmd(targetReader TargetReader, targetWriter TargetWriter, configRe
 				if err != nil {
 					return err
 				}
+			case "namespace":
+				err := setKubectlNamespace(args[1])
```

Comment:
Also validate args[1] here, in case you enter `g target namespace` you will otherwise get a panic.